src/core/starboard/config.rs

```diff
@@ -93,7 +93,7 @@ impl StarboardConfig {
                 emojis.extend(sb.settings.downvote_emojis);
 
                 let configs = StarboardOverride::list_by_starboard(&bot.pool, sb.id).await?;
-                for c in configs {
+                if let Some(c) = configs {
```

Comment:
This won't iterate over configs?

Reply:
The type of `configs` is `Option<StarboardOverride>`, not `Vec<_>`. And iterating over `Option` is same as doing `if let`.

Reply:
Well that's a bug, it should be returning a list of StarboardOverride.

Reply:
I pushed a fix to main, if you merge that into this branch this can be changed back to an iteration.

Also seems weird that you can iterate over an option tbh.

Reply:
ok, done